core/src/test/java/org/elasticsearch/script/expression/ExpressionScriptTests.java

```diff
@@ -385,8 +388,63 @@ public void testStringSpecialValueVariable() throws Exception {
             message = e.toString();
         }
         assertThat(message + "should have contained ExpressionScriptExecutionException",
-                   message.contains("ExpressionScriptExecutionException"), equalTo(true));
+                message.contains("ExpressionScriptExecutionException"), equalTo(true));
         assertThat(message + "should have contained text variable error",
-                   message.contains("text variable"), equalTo(true));
+                message.contains("text variable"), equalTo(true));
+    }
+
+    public void testExecutableScripts() throws Exception {
```

Comment:
@colings86 This was perfect.  Thanks. @rjernst I believe I have addressed all your concerns related to testing by adding cases for aggregations, mapping, and update scripts using expressions.